public/components/trace_analytics/components/common/helper_functions.tsx

```diff
@@ -582,3 +589,56 @@ export const generateServiceUrl = (service: string, dataSourceId: string) => {
 
   return `${url}&datasourceId=`;
 };
+
+interface FullScreenWrapperProps {
+  children: React.ReactNode;
+  onClose: () => void;
+  isFullScreen: boolean;
+}
+
+export const FullScreenWrapper: React.FC<FullScreenWrapperProps> = ({
+  children,
+  onClose,
+  isFullScreen,
+}) => {
+  if (!isFullScreen) return <>{children}</>;
+
+  return (
+    <EuiOverlayMask>
+      <div
+        style={{
+          position: 'fixed',
+          top: 0,
+          left: 0,
+          width: '100vw',
+          height: '100vh',
+          backgroundColor: '#fff',
+          zIndex: 9999,
+          display: 'flex',
+          flexDirection: 'column',
```

Comment:
Can add a class here and move the custom styling to `index.scss`

Reply:
Added classes and move to scss file
```
.full-screen-wrapper {
  position: fixed;
  top: 0;
  left: 0;
  width: 100vw;
  height: 100vh;
  background-color: #fff;
  z-index: 9999;
  display: flex;
  flex-direction: column;
}

.full-screen-close-icon {
  position: absolute;
  top: 4px;
  right: 4px;
  z-index: 10000;
}

.full-screen-content {
  flex: 1 1 auto;
  overflow: auto;
}
```